api/cluster/templater_test.go

```diff
@@ -90,9 +90,13 @@ var (
 	}
 
 	standardTransformerConfig = config.StandardTransformerConfig{
-		ImageName:       "merlin-standard-transformer",
-		FeastServingURL: "serving.feast.dev:8081",
-		FeastCoreURL:    "core.feast.dev:8081",
+		ImageName:                   "merlin-standard-transformer",
+		DefaultFeastServingEndpoint: "serving.feast.dev:8081",
```

Comment:
Can we be more consistent on choosing name? I prefer to use `DefaultFeastServingURLs`